auth-web/src/stores/org.ts

```diff
@@ -159,6 +167,10 @@ export const useOrgStore = defineStore('org', () => {
     return [AccessType.GOVN, AccessType.GOVM].includes(state.originAccessType as AccessType)
   })
 
+  const isGovmOrg = computed<boolean>(() => {
```

Comment:
Might need to check this, could be computing for ACTIVE instead of org in creation